README.md

```diff
@@ -44,214 +44,102 @@ information on its dependencies.
 * [BridgeStan](https://github.com/roualdes/bridgestan)  ([BSD-3
 licensed](https://opensource.org/license/bsd-3-clause))
 
+## Using WALNUTS in a C++ project
 
-## CMake Preliminary Build
+This library is header only, and only requires Eigen (also header only)
+for downstream usage. If your project uses CMake, you can depend on our
+`walnuts` library target. If not, any method of adding the `include/`
+folder of this repository to your build system's include paths should suffice
+as long as you also provide Eigen yourself.
 
-All build steps use CMake. The root configuration file is:
+## Building the examples and tests
 
-* `CMakeLists.txt`
+CMake is required to build the examples and tests.
 
-All instructions assume a `bash` shell (typically at `/usr/bin/bash`)
-and that commands are run from the top-level `walnuts` directory of
-the repository.
+### Configuring the build
 
-### Internet Connection Required
+The basic configuration is
 
-The build uses CMake's `FetchContent` and `ExternalProject_Add` to
-manage dependencies. **An internet connection is required when first
-building the project.**
-
-### Initial Configuration
-
-**Before building any targets**, you must configure the build with:
-
-```bash
-cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
+```sh
+cmake <options> <repo_root>
```

Comment:
This would have confused me before.  I find it the most awkward part of Cmake compared to make other than the incredible verbosity.